<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>365</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>799</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>799</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>User Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BBIPED Configuration </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ConfigurationTab">
      <attribute name="title">
       <string>Solver Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Serial Solver</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="customizedSolverLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Choose Serial Solver Engine location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="customizedSolverEnginePath">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="customizedSolverEngineToolButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearSolverEngineLocationButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/messageBoxIcons/images/cross2.png</normaloff>:/messageBoxIcons/images/cross2.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Parallel Solver Configuration</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Choose Parallel Partition Engine location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="partitionFolderlineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="parallelPartitiontoolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearPartitionLocationButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/messageBoxIcons/images/cross2.png</normaloff>:/messageBoxIcons/images/cross2.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Choose Parallel Solver Engine Location </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="parallelEngineFolderlineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="parallelEnginetoolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearParallelEngineLocationButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/messageBoxIcons/images/cross2.png</normaloff>:/messageBoxIcons/images/cross2.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="meshViewConfigTab">
      <attribute name="title">
       <string>Mesh viewer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Set up the Mesh Viewer Configuration Path</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="meshViewerConfLineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="meshViewerConfLineToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteMeshViewerConfButton">
           <property name="toolTip">
            <string>Delete the path</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/messageBoxIcons/images/cross2.png</normaloff>:/messageBoxIcons/images/cross2.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PostProcessorConfigTab">
      <attribute name="title">
       <string>PostProcessor viewer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="paraviewTitle1">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Set up the PostProcessing Viewer Path. </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ParaviewExplanationLabel2">
         <property name="text">
          <string>We strongly recommend the usage of stable versions of Paraview</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="postProcessorViewerConfLineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="postProcessorViewerConfLineToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deletePostProcessorViewerConfButton">
           <property name="toolTip">
            <string>Delete the path</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/messageBoxIcons/images/cross2.png</normaloff>:/messageBoxIcons/images/cross2.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="warningTab">
      <attribute name="title">
       <string>Warning Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="warningAppCheckBox">
           <property name="text">
            <string>Show Warning message when the BBIPED  platform is closed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="warningProjectCheckBox">
           <property name="text">
            <string>Show Warning message when a project is closed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="templateConfTab">
      <attribute name="title">
       <string>Template Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Choose the template for your configuration file views for all  your sessions. 
 You will have it available under the personalised view</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Template for the configuration file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="templateConfLineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="templateConfToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteTemplateButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../toolbarIcons.qrc">
             <normaloff>:/new/toolbarIcons/images/cross.png</normaloff>:/new/toolbarIcons/images/cross.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="resetPreferencesToDefaultButton">
       <property name="text">
        <string>Reset Preferences to default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="preferenceButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../toolbarIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>preferenceButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preferenceButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
